src/app/layout.tsx

```diff
@@ -15,6 +15,7 @@ const poppins = Poppins({
   subsets: ['latin'],
   weight: ['900', '800', '700', '600', '500', '400', '100'],
   style: ['normal'],
+  variable: '--poppins-font',
 });
 
```

Comment:
Using the `variable` attribute means the `poppins` object will now expose a relevant `className` through the `poppins.variable` attribute. Attach that as a `className` to `html` and remove the manual variable setup in the `globals.css` file.